bokehjs/src/lib/client/connection.ts

```diff
@@ -21,28 +23,25 @@ export class ClientConnection {
   closed_permanently: boolean = false
 
   protected _current_handler: ((message: Message) => void) | null = null
-  protected _pending_ack: [(connection: ClientConnection) => void, Rejecter] | null = null // null or [resolve,reject]
-  protected _pending_replies: {[key: string]: [(message: Message) => void, Rejecter]} = {} // map reqid to [resolve,reject]
+  protected _pending_replies: { [key: string]: [MessageResolver, Rejecter] } = {}
```

Comment:
On the stylistic level we don't use spaces in this context, so the original is preferable. However, given the extent of changes to this module, let's switch from using plain objects as maps and use `Map<K, V>` instead.

Reply:
Sure.
Heh, after seeing the linter complaining so often, I started to trust it too much.
Is it possible to add the rule about spaces to the linter as well?

Reply:
Yes, but I've been postponing such rules (and especially their fixes) till I get to splitting up bokehjs, which will introduce enough churn to allow this.